docs/basics/functions.md

```diff
@@ -665,6 +669,109 @@ Which would return the following:
 "metrics": []
 ```
 
+### sample_include_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API | only creating the sample if both a key and value is found in the sample |
+|| in absence of sample_include_filter, it defaults to include all|
+|| if the sample cannot pass sample_include_filter filter, the sample_filter and sample_exclude_filter will not be evalucated|
```

Comment:
small typo here **evalucated** should be **evaluated**

Reply:
Updated. thanks!